IEEE-PSH/hackpsh-web
# Admin Controls for Countdown

# Description
Added a new page for users with `admin` role. Currently contains "Event Details" which allows administrators to set the date, time, and duration of the event. The countdown component now reflects these settings which are stored in a new table called `app_event`. To accomplish this, such changes were made:

- Added `app_event` table to store event details (ex: date, start time, end time, duration).
- Added sql line to enable RLS on app_event.
- Added procedures to get/modify data from this table.
- Added new schemas for event details form and server procedures.
- Added "administration" route for admin controls.
- Added dropdown link to "administration".

# Test Guidelines
1. Log in.
2. Change role directly in `app_user_profile` to `admin`.
3. Click dropdown.
4. Click "Administration".
5. Submit form changes.
6. View Home to see updated countdown.
